PLATER/requirements.txt

```diff
@@ -5,11 +5,9 @@ pytest==6.2.4
 pytest-asyncio==0.15.1
 starlette==0.14.2
 uvicorn==0.14.0
-git+https://github.com/ranking-agent/reasoner-transpiler.git@main#egg=reasoner
+reasoner-transpiler==1.8.0
 reasoner-pydantic==1.1.2.1
 httpx==0.18.2
 pytest-httpx==0.12.0
-bmt==0.7.2
```

Comment:
Since we import bmt(-lite) directly in Plater, I would include bmt-lite-2.1.0 as an explicit dependency here. The version will have to be consistent with the range required by reasoner-transpiler. If reasoner-transpiler upgraded to a version of bmt-lite that is incompatible with the way Plater uses it, we'd like to catch it when installing dependencies rather than at runtime. Or if, for some reason, a later reasoner-transpiler stopped using bmt entirely, we wouldn't want to lose it.

Reply:
added bmt-lite-2.1.0==2.1.1 to requirements file